defaults/main.yml

```diff
@@ -2,9 +2,9 @@
 ### ManageIQ/CloudForms ###
 
 # QCOW2 ManageIQ/CloudForms image URL:
-miq_qcow_url: http://releases.manageiq.org/manageiq-ovirt-fine-1.qc2
-miq_image_path: /tmp/ovirt_image_data
-miq_qcow_checksum: sha256:b3644e8ac75af9663d19372e21b8a0273d68e54bfd515518321d3102d08daebd
+miq_qcow_url: http://releases.manageiq.org/manageiq-ovirt-gaprindashvili-1.qc2
+miq_image_path: ./manageiq-ovirt-gaprindashvili-1.qc2
```

Comment:
Is it wise to store to the current directory where the role is executed from? Shouldn't we use `/tmp/manageiq-ovirt-gaprindashvili-1.qc2` instead?

Reply:
A predictable filename on /tmp is never a good idea. We could use a temp file (I think that's the right thing to do actually).

Reply:
The reason I've used predictable name is to not always re-run download in case of failure and re-run. If we will create a temporary file we will loose the name for the second run and we must download again.

Reply:
Indeed - and that's exactly why I've left it in the current directory. It's safer.

Reply:
Ondro, would it work under AWX? No idea what's current directory there ...